tox.ini

```diff
@@ -9,3 +9,10 @@ deps =
 
 commands =
     py.test
+
+[testenv:docs]
+basepython=python
+changedir=docs
+deps=sphinx
+commands=
+sphinx-build -b html -d {envtmpdir}/doctrees docs {envtmpdir}/html
```

Comment:
I believe the build is failing because line 18 needs to be indented, but I could be wrong. @skarbot, do you know?